core/src/components/modal/modal.tsx

```diff
@@ -140,6 +142,13 @@ export class Modal implements ComponentInterface, OverlayInterface {
    */
   @Prop() handle?: boolean;
 
+  /**
+   * The interaction behavior for the sheet modal when the handle is pressed.
+   * Defaults to `"none"`, which  means the modal will not change size or position when the handle is pressed.
+   * Set to `"cycle"` to let the modal cycle between available breakpoints when pressed.
```

Comment:
Can we add a note to state that the behavior does not apply if `handle` is `false` or when the sheet modal is not used?